src/braket/circuits/gates.py

```diff
@@ -79,11 +107,11 @@ def h(target: QubitSetInput) -> Iterable[Instruction]:
 Gate.register_gate(H)
 
 
-class I(Gate):  # noqa: E742, E261
+class I(_FiniteOrderGate):  # noqa: E742, E261
```

Comment:
Isn't `I` hermitian ?

Reply:
`I` has order 1, which implies Hermiticity, but here we do one better by simply not adding a gate at all, as opposed to a single gate. I don't have a problem with making it explicitly Hermitian.

Reply:
It's this kind of discussion that makes me not want to have _HermitianGate in the first place; I don't think it adds too much value and just makes development slower.